app/server/controllers/login.ts

```diff
@@ -142,7 +142,7 @@ function getIdamCallback(
           }
         } else {
           const hearingsByName = getHearingsByName(hearings);
-
+          AppInsights.trackTrace(`[Cases count ${hearings.length}] - User logged in successfully as ${email}`);
```

Comment:
What does 'Cases count {hearings length}' mean?

Reply:
Its list of cases, in MYA user either can land on only one case by providing the tya number or  on summary page without tya which shows list of cases

Reply:
but why is it looking at the hearing count?

Reply:
hearing object is the case details in MYA